src/components/signup-view/signup-view.jsx

```diff
@@ -0,0 +1,72 @@
+import { useState } from "react";
+
+export const SignupView = () => {
+  const [username, setUsername] = useState("");
+  const [password, setPassword] = useState("");
+  const [email, setEmail] = useState("");
+  const [birthday, setBirthday] = useState("");
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+
+    const data = {
+      Username: username,
+      Password: password,
+      Email: email,
+      Birthday: birthday
+    };
+
+    fetch("https://myflix-ghibli-7c8d5913b80b.herokuapp.com/users", {
+      method: "POST",
+      body: JSON.stringify(data),
+      headers: {
+        "Content-Type": "application/json"
+      }
+    }).then((response) => {
+      if (response.ok) {
+        alert("Signup successful!");
+        window.location.reload();
+      } else {
+        alert("Signup failed")
+      }
+    })
+  };
+
+  return (
+    <form onSubmit={handleSubmit}>
+      <label>
+        Username:
+        <input type="text"
+        value={username}
+        onChange={(e) => setUsername(e.target.value)}
+        minLength="3"
+        required />
+      </label>
+      <label>
+        Password:
+        <input 
+        type="password"
+        value={password}
+        onChange={(e) => setPassword(e.target.value)}
+        required />
+      </label>
+      <label>
+        Email:
+        <input 
+        type="email"
+        value={email}
+        onChange={(e) => setEmail(e.target.value)}
+        required />
+      </label>
+      <label>
+        Birthday:
+        <input 
+        type="date"
+        value={birthday}
+        onChange={(e) => setBirthday(e.target.value)}
```

Comment:
if you end up saving as date, instead of passing e.target.value, you can pass `new Date(e.target.value)`. You made need to do additional validation to get this to work, but would be a better UX